public/api/v1/openapi.yml

```diff
@@ -836,8 +836,6 @@
         properties:
           name:
             type: "string"
-          account:
```

Comment:
The `DigitalServiceAccount` type doesn't have a `service` field. Should that be added?

Reply:
added